stabilizer/src/main/java/com/hazelcast/stabilizer/agent/workerjvm/WorkerJvmLauncher.java

```diff
@@ -124,10 +130,27 @@ private WorkerJvm startWorkerJvm(String mode) throws IOException {
         new WorkerJvmProcessOutputGobbler(process.getInputStream(), new FileOutputStream(logFile)).start();
         workerJvm.process = process;
         workerJvm.mode = WorkerJvm.Mode.valueOf(mode.toUpperCase());
+        copyResourcesToWorkerId(workerId);
         workerJVMs.put(workerId, workerJvm);
         return workerJvm;
     }
 
+    private void copyResourcesToWorkerId(String workerId) throws IOException {
+        final String testSuiteId = agent.getTestSuite().id;
+        if (!new File(WORKERS_PATH + "/" + testSuiteId + "/upload/").exists()) {
```

Comment:
Can you explain what you mean? And why do you think that 

```
File uploadDirectory= new File(WORKERS_PATH + "/" + testSuiteId + "/upload/");
if (!uploadDirectory.exists()) {
}
```

Is any different then

```
if (!new File(WORKERS_PATH + "/" + testSuiteId + "/upload/").exists()) {
}
```
From an exception point of view?